### PR TITLE
[Artwork] Add remaining OtherWorks grids

### PR DESCRIPTION
**Pull request by damassi**
Adds two grids for contexts `ArtworkContextFair` and `ArtworkContextPartnerShow`. 

Follow-up prs: 
- include `exclude` variables for excluding the current artwork from the grid set
- visual cleanup / code cleanup

### CODE REVIEW

File: src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx [hunk 1]
```diff
@@ -69,10 +69,10 @@ export const ArtworkContextAuction: React.SFC<{
     return <AuctionArtworkGrid artwork={props.artwork} />
   } else {
     return (
-      <>
+      <Join separator={<Spacer my={2} />}>
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx]: oh yea, `Join`!

**Comment by mzikherman** [comment]
Looks great.

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit dfcfd185bbc08d1a76ad1aa8d4a2a6346b82f962

https://deploy-preview-1616--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 13efffc1b80be8250a5c68f8feb9eb531bd5b99a

https://deploy-preview-1616--artsy-reaction.netlify.com

**Pull request merged**